### PR TITLE
IGAPP-540: Display Pois on Map - Native

### PR DESCRIPTION
**Pull request by Taggotty**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

**Review by f1sh1918 — APPROVED** [review]

**Review by klinzo — APPROVED** [review]
Tested on Android :+1: 

Probably not related to your issue:
After switching the language in the pois route, I got the following error (among others):
> Mapbox error You're calling `queryRenderedFeatures` after the `MapView` was destroyed, were you invoking it after `onDestroy()`? {"level": "error", "message": "You're calling `queryRenderedFeatures` after the `MapView` was destroyed, were you invoking it after `onDestroy()`?", "tag": "Mbgl-NativeMapView"}

Same for methods `moveBy`, `getCameraValues` and `setGestureInProgress`. However, I could not reproduce it afterwards.

File: native/src/components/MapView.tsx [hunk 1]
```diff
@@ -15,11 +15,28 @@ const StyledMap = styled(MapboxGL.MapView)`
...
+    iconAllowOverlap: true,
+    iconIgnorePlacement: true,
+    iconImage: ['get', 'symbol'],
+    textField: ['get', 'title'],
+    textFont: ['Roboto Regular'],
+    textOffset: [0, textOffsetY],
+    textAnchor: 'top',
+    textSize: 12
+  }
 }
```
Comment (klinzo) [inline native/src/components/MapView.tsx:35]: Just for my understanding: Is this something that will be replaced at a later point by the style spec served from our servers in this issue? Or will this stay? https://issues.tuerantuer.org/browse/IGAPP-535

Comment (maxammann) [reply native/src/components/MapView.tsx:35]: Could be, this is open for discussion I'd say. By moving this into the style.json we could reduce code duplication if we show the map in native and web.

**Review by maxammann — APPROVED** [review]

File: native/src/components/MapView.tsx (hunk 1 above)
Comment (Taggotty) [reply native/src/components/MapView.tsx:35]: I'd say it is better to share what is the same. I'am not sure about the fonts though. As we use different fonts in native and web right?

**Comment by Taggotty** [comment]
> Tested on Android 
> 
> Probably not related to your issue:
> After switching the language in the pois route, I got the following error (among others):
> 
> > Mapbox error You're calling `queryRenderedFeatures` after the `MapView` was destroyed, were you invoking it after `onDestroy()`? {"level": "error", "message": "You're calling `queryRenderedFeatures` after the `MapView` was destroyed, were you invoking it after `onDestroy()`?", "tag": "Mbgl-NativeMapView"}
> 
> Same for methods `moveBy`, `getCameraValues` and `setGestureInProgress`. However, I could not reproduce it afterwards.

Could not reproduce for now.

**Pull request merged**